common/lib/capa/capa/templates/jsinput.html

```diff
@@ -1,11 +1,12 @@
+<%page expression_filter="h"/>
 <%! from openedx.core.djangolib.markup import HTML %>
-<section id="inputtype_${id}" class="jsinput"
+<div id="inputtype_${id}" class="jsinput"
     data="${gradefn}"
     % if saved_state:
-    data-stored="${saved_state|x}"
+    data-stored="${saved_state}"
```

Comment:
@robrap Do you foresee any issues with changing from XML-escaping to HTML-escaping?